utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
This would send all `stderr` from our shimmed application as `DebugLevel` and all `stdout` from our shimmed application as `Info`? That means that applications would effectively drop everything coming out of `stderr` in typical production environments, which seems unwise. If we're going to assign log-levels at all, shouldn't `stderr` be at a higher-priority level?